packages/client/src/clients/guide/client.ts

```diff
@@ -156,7 +157,7 @@ export class KnockGuideClient {
   // to resolve and render the prevailing one.
   private stage: GroupStage | undefined;
 
-  private intervalId: ReturnType<typeof setInterval> | undefined;
+  private counterIntervalId: ReturnType<typeof setInterval> | undefined;
```

Comment:
Addressing this feedback: https://github.com/knocklabs/javascript/pull/668/files#r2246072354